README.md

```diff
@@ -455,10 +455,19 @@ via `--retry-count` flag.
 
 ## Using wildcards
 
-Most shells can attempt to expand wildcards before passing the arguments to
-`s5cmd`, resulting in surprising `no matches found` errors.
+One some shells, like zsh, the `*` character gets treated as a file globbing
+wildcard, which causes unexpected results for `s5cmd`. You might see an output
+like:
 
-To avoid this problem, surround the wildcarded expression with single quotes.
+```
+zsh: no matches found
+```
+
+If that happens, you need to wrap your command in single quotes, like:
+
+```
+s5cmd cp *.gz s3://bucket/
```

Comment:
Single quotes are missing in the example :)

Reply:
done :)